src/OrchardCoreContrib.ContentLocalization/Extensions/LiquidFilterServicesExtensions.cs

```diff
@@ -0,0 +1,14 @@
+using Microsoft.Extensions.DependencyInjection;
+using OrchardCore.Liquid;
+using OrchardCoreContrib.ContentLocalization.Liquid;
+
+namespace OrchardCoreContrib.ContentLocalization.Extensions;
+
+public static class LiquidFilterServicesExtensions
+{
+    public static void AddLiquidFilters(this IServiceCollection services)
+    {
+        services.AddLiquidFilter<CyrToLatFilter>("cyr_to_lat");
```

Comment:
Before I go further, it would be nice to have such thing as feature that could be on and off. It should rely on Liquid module

Reply:
I don't quite understand, can you give an example or describe it in more detail?

Reply:
Have a look at the `OCC.Users` module it contains many features https://github.com/OrchardCoreContrib/OrchardCoreContrib.Modules/blob/main/src/OrchardCoreContrib.Users/Manifest.cs each one has its own startup to register its own service, similarly you can add the liquid filter as a feature

Reply:
@hishamco Got it, thanks, done.

Reply:
Looks great, just let me try it then merge

Reply:
BTW you can demo it next week in Orchard Meeting if you are interested

Reply:
Sorry for late. Thank you for the invitation, but I have to decline, I don't have a good knowledge of spoken English, I think it will be difficult for me to carry on a conversation :)

Reply:
> I don't have a good knowledge of spoken English

Me too :) coz I'm not a native speaker, but I'm struggling